eras/conway/impl/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Version history for `cardano-ledger-conway`
 
+## 1.17.1.0
+
+*
+
+### `testlib`
+
+* Move `TxInfo` golden tests over from the older `-test` package. #4599
+  * Also move the `gen-golden` executable over.
+
 ## 1.17.0.0
 
```

Comment:
How about this big fat bold note in the PR template that you haven't checked :wink: 

![image](https://github.com/user-attachments/assets/e127f520-87cc-4aaa-9643-189aa72bfe1e)

1.17.0.0 hasn't been released yet!

Reply:
Thanks Alexey! 🙂